decidim-comments/app/events/decidim/comments/comment_created_event.rb

```diff
@@ -34,7 +34,9 @@ def notification_title
           "decidim.comments.events.comment_created.#{comment_type}.notification_title",
           resource_title: resource_title,
           resource_path: resource_locator.path(url_params),
-          author_name: comment.author.name
+          author_nickname: comment.author.nickname,
+          author_name: comment.author.name,
+          author_path: Decidim::UserPresenter.new(comment.author).profile_path
```

Comment:
Other minor nit, you could instantiate a single presenter and call all user methods on it. You'll save one "@" in the email copy because the nickname will be ready for presentation.